src/test/scala/edu/ie3/simona/model/participant/BmModelSpec.scala

```diff
@@ -39,7 +39,9 @@ class BmModelSpec extends UnitSpec {
     isCostControlled = true,
     EUR(0.05),
     EuroPerKilowatthour(0.51d),
+    EuroPerKilowatthour(0.51d),
```

Comment:
Maybe pick a different value for feedInTariff than opex? In the best case, something that is at least in the ballpark of something realistic